Weather App/UI/SettingsVC/SettingsData.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  SettingsData.swift
+//  Weather App
+//
+//  Created by Bruno Benčević on 9/7/21.
+//
+
+import Foundation
+
+struct SettingsData: Codable {
+    var useCelsius: Bool
```

Comment:
Do properties need to be `var`s?

Reply:
While keeping track of settingsData for the settings screen, the viewModel's callbacks must be able to update those since they are saved when closing the screen.